src/styles/main.scss

```diff
@@ -0,0 +1,6 @@
+@import 'variables';
+@import 'header';
+@import 'index';
+@import 'stars';
+@import 'card';
+@import 'reset';
```

Comment:
The 'reset' stylesheet should typically be imported before other stylesheets to ensure that the reset styles are applied first. Consider moving the '@import 'reset';' line to the top of the imports.